backend/services/user_groups.go

```diff
@@ -79,9 +79,13 @@ func (i CreateUserGroupInput) validateUserGroupInput() error {
 func AddUsersToGroup(tx *database.Transactable, userSlugs []string, groupID int64) error {
```

Comment:
@JoelAtDeluxe Initially I improved this function by iterating through each userSlug and performing an upsert, but here's my attempt to reduce it to one query. It looks a little janky, but (in theory) should be much more performant that inserting each userSlug individually. Any way you'd improve this?

Reply:
I think there is a better way, but we  haven't leveraged this in the code before. I'll post a suggestion  with some details.